cachalot/utils.py

```diff
@@ -208,6 +223,12 @@ def __update_annotated_subquery(_annotation: Subquery):
         sql = query.get_compiler(db_alias).as_sql()[0].lower()
         tables = _get_tables_from_sql(connections[db_alias], sql)
 
+    # Additional check of the final SQL.
+    # Potentially overlooked tables are added here. Tables may be overlooked by the regular checks
+    # as not all expressions are handled yet. This final check acts as safety net.
+    final_check_tables = _get_tables_from_sql(connections[db_alias], str(query), enable_quote=True)
```

Comment:
difference between this and L224? I say just put in try clause.

Reply:
> difference between this and L224? I say just put in try clause.

Put it into an try - except - **else**.